pkgs/applications/networking/zerobin/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, nodePackages
+, doit
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zerobin";
+  version = "1.0.5";
+  src = fetchFromGitHub {
+    owner = "Tygs";
+    repo = "0bin";
+    rev = "v${version}";
+    sha256 = "1dfy3h823ylz4w2vv3mrmnmiyvf6rvyvsp4j3llr074w9id0zy16";
+  };
+
+  disabled = python3Packages.pythonOlder "3.7";
+
+  nativeBuildInputs = with python3Packages; [ doit pyscss nodePackages.uglify-js ];
```

I would not import `doit` from `all-packages.nix` and use `doit` right after `with python3Packages`, although the result is similar. And to make that even clearer, I would put here:

```suggestion
  nativeBuildInputs = [
    python3Packages.doit
    python3Packages.pycss
    nodePackages.uglify-js
  ];
```

---

Done, thanks.